clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureServerGroupResourceTemplate.groovy

```diff
@@ -458,6 +458,23 @@ class AzureServerGroupResourceTemplate {
     }
   }
 
+  static class ScaleSetOsProfileWindowsConfiguration extends ScaleSetOsProfileProperty implements ScaleSetOsProfile {
+    OsProfileWindowsConfiguration windowsConfiguration
+
+    ScaleSetOsProfileWindowsConfiguration(AzureServerGroupDescription description) {
+      super(description)
+      windowsConfiguration = new OsProfileWindowsConfiguration(description)
+    }
+  }
+
+  static class OsProfileWindowsConfiguration {
+    String timeZone
+
+    OsProfileWindowsConfiguration(AzureServerGroupDescription description) {
+      timeZone = description.timeZone
```

Comment:
@EasyAsABC123 lets us chat on this , i feel that if timezone is the only factor that can be used to decide if this is a windows config we should probably also rename the attribute to reflect that this is a specific to windows ( for example: description.windowsConfig.timeZone)

Reply:
```
"windowsConfiguration": {
        "provisionVMAgent": "boolean",
        "enableAutomaticUpdates": "boolean",
        "timeZone": "string",
        "additionalUnattendContent": [
          {
            "passName": "OobeSystem",
            "componentName": "Microsoft-Windows-Shell-Setup",
            "settingName": "string",
            "content": "string"
          }
        ],
        "winRM": {
          "listeners": [
            {
              "protocol": "string",
              "certificateUrl": "string"
            }
          ]
        }
      },
```

Reply:
Right, so these are specific to windowsConfig. In future we may need to add other config as well so it would help group it under a sub bean class

Reply:
after discussion we decided to migrate this to `windowsTimeZone`